ykrt/pass_finder/try_passes.py

```diff
@@ -185,18 +211,114 @@ def binary_split(logf, passes, is_prelink):
         print(72 * "=")
         print(list_of_passes_to_str(ok_passes))
 
-def main(logf, is_prelink):
+def evaluate_fitness(glogf, is_prelink, entity, passes, cwd):
+    try_passes = []
+    for i, bit in enumerate(entity):
```

Comment:
I think this can become something like:

```python
try_passes = [passes[i] for (i, x) in enumerate(entity) if x]
```

Warning: I might be wrong!

Reply:
Done: 6c3f62f2dd481f0a792a6e681a1d4f0fe6b336fc